RELEASE/scripts/autoscend/paths/legacy_of_loathing.ash

```diff
@@ -41,8 +41,11 @@ boolean lol_buyReplicas()
 		}
 		if(contains_text(page, "patriotic eagle")) //2023
 		{
-			buy($coinmaster[Replica Mr. Store], 1, $item[replica sleeping patriotic eagle]);
-			use(1, $item[replica sleeping patriotic eagle]); // put in terrarium
+			if(!is100FamRun()) //If this isn't a 100% familiar run, go ahead and get another familiar
+				{	
+					buy($coinmaster[Replica Mr. Store], 1, $item[replica sleeping patriotic eagle]);
+					use(1, $item[replica sleeping patriotic eagle]); // put in terrarium
+				}
 		}
 		else if(contains_text(page, "<b>2004</b>"))
```

Comment:
This is the bug I found in my initial commit.

If we found the Eagle on the page, we'd enter the condition statement on line 44, , check to see if it was a 100% Fam run, then do nothing. The problem is that in a 100% Fam run, we always find the eagle, and we always do nothing. 

This won't trigger the 'Else If' to look for the text 2004 on line 51, causing the entire script to halt (an issue I saw in testing). We need to check to see if this is a 100% Fam Run first, then ignore the Eagle the rest of the run.

Note: This assumes that the Adventurer has already purchased up to the familiar they want to do the run with. There is no code in the script to support purchasing a familiar for 100% runs on turn 0. (that would require a lot of error checking to ensure you have enough Replica Mr. A's to buy your grey gosling on turn 0 [19 required I think])